test/e2e/workable.go

```diff
@@ -0,0 +1,300 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"os/exec"
+	"runtime"
+	"strings"
+	"testing"
+
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+)
+
+func testWorkable(ctx context.Context, kwokctlPath, name string) error {
+	cmd := exec.Command("kubectl", "config", "current-context")
```

Comment:
> workable.go:236: exec: "kubectl": executable file not found in $PATH

https://github.com/kubernetes-sigs/kwok/actions/runs/10561061180/job/29255960861?pr=1195


Add `${rootDir}/bin` to $PATH in `main_test.go`, I remember kubectl will download there.